src/content/docs/tutorial-create-alerts/create-chart-alerts.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: "Create an alert from a chart"
+metaDescription: "How to create an alert from a chart with New Relic"
+---
+
+import alertsCreate from 'images/alerts_screenshot-crop_create-query.webp'
+
+import alertsSignal from 'images/alerts_screenshot-crop_signal-select.webp'
+
+import alertsTune from 'images/alerts_screenshot-crop_signal-tune.webp'
+
+import alertsThresholds from 'images/alerts_screenshot-crop_set-thresholds.webp'
+
+import alertsDetails from 'images/alerts_screenshot-crop_condition-details.webp'
+
+Using a chart is one of the easiest ways to set up a New Relic alert condition. Instead of creating a NRQL query from scratch, charts let you use pre-existing NRQL conditions to begin the process. In this example, we'll set up an alert condition using an error response chart.
+
+<img
+    title="Create chart alert"
+    alt="A screenshot displaying where to create an alert condition from a chart in New Relic"
+    src={alertsCreate}
+/>
+
+<Callout variant="tip">
+If you need more information on any of the options or terms covered in the steps below, check out our [alerts docs](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts/).
+    </Callout>
+
+<Steps>
+    <Step>
+## Select a chart and define your signal
+
+<img
+    title="Defining a signal"
+    alt="A screenshot detailing how to select a query for an alert condition"
+    src={alertsSignal}
+/>
+
+1. From any **Dashboard**, select the chart you want to create the query from, then select **Create alert condition**
+
+2. Select the signal you want to use to create the alert condition. To keep things simple, select the **Build a query** option for your first condition
+
+3. Modify the NRQL query to match what you want you to monitor, if necessary. For now, we recommend leaving the query as is, but once you have a better handle on alerts and NRQL queries, you can change the query to monitor nearly any data threshold
+
+4. Select **Next**
+    </Step>
+    <Step>
+## Adjust the signal behavior
+
+<img
+    title="Tuning your signal"
+    alt="A screenshot displaying how to set options for a signal in an alert condition"
+    src={alertsTune}
+/>
+
+1. Set the data aggregation options. You will need to:
+    * Use the **Window duration** setting to determine how frequently we aggregate your data, such as groupings of every 5 minutes or every hour
```

Comment:
From my review the SME, we thought this one would be hard to make a recommendation for because it's so variable on what the user is trying to monitor. I can add in a "if you're unsure, just leave it at the default setting" kind of line to it, but avoided doing so in drafting because I was afraid of causing confusion.